allensdk/brain_observatory/gaze_mapping/__main__.py

```diff
@@ -0,0 +1,252 @@
+import h5py
+import logging
+import sys
+
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+
+from argschema import ArgSchemaParser
+
+from allensdk.brain_observatory.argschema_utilities import (
+    write_or_print_outputs
+)
+
+from ._schemas import InputSchema, OutputSchema
+from ._eye_calibration import compute_ellipse_areas, EyeCalibration
+from ._filter_utils import (post_process_cr,
+                            post_process_areas)
+
+
+def repackage_input_args(parser_args: dict) -> dict:
+    """Repackage arguments obtained by argschema.
+
+    1) Converts individual coordinate/rotation fields to numpy
+       position/rotation arrays.
+    2) Converts path strings to pathlib Paths
```

Comment:
note for future: you can do this by making a [marshmallow field](https://marshmallow.readthedocs.io/en/stable/_modules/marshmallow/fields.html) and telling it how to deserialize.